openedx/features/learner_analytics/templates/learner_analytics/dashboard.html

```diff
@@ -48,19 +48,26 @@
     </header>
     <div class="page-content learner-analytics-dashboard-wrapper">
         <div class="learner-analytics-dashboard">
```

Comment:
FYI: @MatthewPiatetsky: I left this class for audit users because I assume this is what you are using to add upsell.

Reply:
thanks, not actually adding upselling to that page yet since it did not exist

Reply:
Ok.  Thanks.